src/webapp/components/landing-page-edit-dialog/LandingPageEditDialog.tsx

```diff
@@ -111,11 +114,40 @@ export const LandingPageEditDialog: React.FC<LandingPageEditDialogProps> = props
     };
 
     const handleFileUpload = useCallback(
```

Comment:
I would rename it to `handleImageFileUpload` because we are using image properties within